proto/axelar/evm/v1beta1/query.proto

```diff
@@ -129,6 +129,19 @@ message ChainsResponse {
             "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
 }
 
+message CommandRequest {
+  string chain = 1;
+  string id = 2;
```

Comment:
```suggestion
  string id = 2 [ (gogoproto.customname) = "ID" ];
```
forgot to add this before

Reply:
done here [d46836d](https://github.com/axelarnetwork/axelar-core/pull/1869/commits/d46836d96b7584a749811b16d728b3349c6d30af)